src/components/ui/hook-form/buttons/index.tsx

```diff
@@ -25,12 +28,31 @@ const BaseButton = ({
             type={type}
             onClick={onClick}
             style={{ ...buttonsStyle, ...style }}
+            className={className}
         >
             {title}
         </Button>
     );
 };
 
+const BaseButton = styled(BaseButtonBase)`
```

Comment:
Um, well. This is a solution, but are you sure we could not use ConfigProvider from antd for this purpose?

Reply:
I don't understand how I can use the configProvider for the dashed button type. The documentation contains tokens only for Primary